wreq.cabal

```diff
@@ -80,6 +80,7 @@ library
   other-modules:
     Network.Wreq.Internal
     Network.Wreq.Internal.AWS
+    Network.Wreq.Internal.OAuth1
```

Comment:
The module names in this list are sorted. Please add this in the correct lexicographical position.